backend/src/test/java/gov/cdc/usds/simplereport/api/IdentityVerificationControllerTest.java

```diff
@@ -51,6 +52,7 @@
   DemoAuthenticationConfiguration.class,
   DemoExperianService.class,
   DemoOktaRepository.class,
+  DevSecurityConfiguration.class
```

Comment:
So this `DevSecurityConfiguration` class should only be imported in test files? 🤔

Reply:
It is needed for integration tests so that they don't actually try to apply any security rules.